changelog/fragments/1716490302-Load-fleet.ssl.certificate_authorities-from-agent-policy.yaml

```
@@ -0,0 +1,32 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: bug-fix
```

IIUC, mTLS (Agent presenting a certificate to Fleet Server / proxy) is a new feature, so this should probably be `feature` or `enhancement`?
```suggestion
kind: feature
```

---

Issue #2247 is flagged as bug so I put this as bug-fix... this changelog is related to reading Certificate Authorities from fleet policy, the other fragment is for presenting a certificate to Fleet server.

The other fragment is also flagged as bug-fix but that one is pointing to #2248 which is labeled as enhancement

---

Ha, I think both should be enhancements but this is probably best decided by @nimarezainia.

---

Enhancement/Feature is fine